pyproject.toml

```diff
@@ -37,26 +36,42 @@ optionals = ["jsonschema", "napalm", "legacycrypt"]
 [tool.poetry.group.dev.dependencies]
 coverage = "*"
 invoke = "*"
-pylint = "^3.1.0"
+requests_mock = "*"
```

Comment:
is this needed? I can be convinced no harm no foul, but ensuring it is intentional.